src/nl/hannahsten/texifyidea/editor/MathEnvironmentEditor.kt

```diff
@@ -24,18 +25,17 @@ class MathEnvironmentEditor(
         val whitespace = if (newEnvironmentName == "inline") indent.length + 1
         else if (oldEnvironmentName == "inline") 1
         else 0
+
         // Extra white space to be added to the beginning of the new environment, when converting from/to inline.
         val extraWhiteSpace = if (newEnvironmentName == "inline") {
             // Add indentation if indentation of old environment is bigger than the previous line.
             val indentOfPreviousLine = document.lineIndentation(document.getLineNumber(environment.textOffset) - 1)
             if (indentOfPreviousLine.length < indent.length) "\n$indent" else " "
-        }
-        else if (oldEnvironmentName == "inline") {
+        } else if (oldEnvironmentName == "inline") {
```

Comment:
```suggestion
        } 
        else if (oldEnvironmentName == "inline") {
```
same for others

Reply:
I'm confused. You are suggesting two contradictory changes...

Reply:
Two changes? I suggested only one suggestion right?